_posts/blog/2018-11-12/2018-11-12-how-will-the-bch-hard-fork-on-nov-15-affect-openbazaar.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: "How Will The <span class='post-title-extra'>BCH Hard Fork</span> Affect OpenBazaar?"
+layout: post
+date: '2018-11-12 06:30:00 -0300'
+social_title: 'How Will The Bitcoin Cash (BCH) Hard Fork on November 15th Affect OpenBazaar?'
+social_description: 'Here is what you need to know about the Bitcoin Cash hard fork on November 15th as it relates to Openbazaar.'
+---
+
+#### What’s Happening with Bitcoin Cash?
+
+There is a [Bitcoin Cash](https://www.bitcoincash.org/) network upgrade due to complete at 4:40PM UTC on November 15th, 2018. Depending on how the upgrade is accepted by the network, the Bitcoin Cash blockchain could fork into two or more incompatible chains. In order to help OpenBazaar users prepare for the fork, here is some information about the OpenBazaar wallet that can help you decide how to handle any BCH funds that you may have stored there.
```

Comment:
The upgrade begins at that time, not ends. We don't know when it will end.

Reply:
updated to ....network upgrade beginning at 4:40PM